### PR TITLE
Sig-scalability test segregation to allow for assignment and cleaning…

### PR DESCRIPTION
**Pull request by timothysc**
**What this PR does / why we need it**:
e2e test segregation to allow for easier test assignment and general cleanliness. 

**Special notes for your reviewer**:
cleaning per-conversation @gmarek 

**Release note**:

```
NONE
```

/cc @gmarek @wojtek-t @kubernetes/sig-scalability-pr-reviews

### CODE REVIEW

File: test/e2e/example_k8petstore.go [hunk 1]
```diff
@@ -1,187 +0,0 @@
-/*
```
Comment (timothysc) [inline test/e2e/example_k8petstore.go]: talked with @jayunit100 about removing, b/c this test is never run.

**Comment by wojtek-t** [comment]
Nice - I also wanted to do this. Will take a more deeper look later.

File: test/e2e/example_k8petstore.go (hunk 1 above)
Comment (wojtek-t) [inline test/e2e/example_k8petstore.go]: +1

File: test/e2e/perf/logging_soak.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package perf
```
Comment (wojtek-t) [inline test/e2e/perf/logging_soak.go]: This is a logging test. We shouldn't move it to perf.

File: test/e2e/perf/OWNERS [hunk 3]
```diff
@@ -0,0 +1,5 @@
+approvers:
+- gmarek
+- wojtek-t
+reviewers:
+- timothysc
```
Comment (wojtek-t) [inline test/e2e/perf/OWNERS]: You can add us as reviewers too (otherwise all reviews will go to you).

**Review by wojtek-t — COMMENTED** [review]
Some super minor comments.

File: test/e2e/perf/logging_soak.go (hunk 2 above)
Comment (timothysc) [inline test/e2e/perf/logging_soak.go]: The logging sat test is specifically a performance test to vet logging, so I dunno.  Seems more performance oriented then logging behavioral testing.

File: test/e2e/perf/OWNERS (hunk 3 above)
Comment (timothysc) [inline test/e2e/perf/OWNERS]: ohh gosh, yeah I'll fix that.

File: test/e2e/perf/logging_soak.go (hunk 2 above)
Comment (wojtek-t) [inline test/e2e/perf/logging_soak.go]: But this test is owned by logging team (similarly as Node benchmarks are owned by node team). I don't think we should move it here.

**Comment by timothysc** [comment]
@wojtek-t comments addressed.

**Pull request merged**